<launch>

  <!-- add robot 1 -->
  <include file="$(find uuv_pso)/launch/upload_rexrov_pso.launch">
    <arg name="namespace" value="rexrov0"/>
    <arg name="velocity_control" value="false"/>
    <arg name="x" value="25"/>
    <arg name="y" value="0"/>
    <arg name="z" value="-2"/>
  </include>

  <!-- add robot 2 -->
  <include file="$(find uuv_pso)/launch/upload_rexrov_pso.launch">
    <arg name="namespace" value="rexrov1"/>
    <arg name="velocity_control" value="false"/>
    <arg name="x" value="25"/>
    <arg name="y" value="25"/>
    <arg name="z" value="-2"/>
  </include>

  <!-- add robot 3 -->
  <include file="$(find uuv_pso)/launch/upload_rexrov_pso.launch">
    <arg name="namespace" value="rexrov2"/>
    <arg name="velocity_control" value="false"/>
    <arg name="x" value="-25"/>
    <arg name="y" value="-25"/>
    <arg name="z" value="-2"/>
  </include>

  <!-- add robot 4 -->
  <include file="$(find uuv_pso)/launch/upload_rexrov_pso.launch">
    <arg name="namespace" value="rexrov3"/>
    <arg name="velocity_control" value="false"/>
    <arg name="x" value="-25"/>
    <arg name="y" value="0"/>
    <arg name="z" value="-2"/>
  </include>

</launch>